senders/telegram/init.go

```diff
@@ -45,14 +46,23 @@ func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger
 	}
 
 	sender.apiToken = apiToken
+	sender.apiServer = senderSettings["apiServer"]
 	sender.frontURI = senderSettings["front_uri"]
 	sender.logger = logger
 	sender.location = location
 	var err error
-	sender.bot, err = telebot.NewBot(telebot.Settings{
-		Token:  sender.apiToken,
-		Poller: &telebot.LongPoller{Timeout: pollerTimeout},
-	})
+	if sender.apiServer == "" {
```

Comment:
Hi @mixazya! Thank you for your PR! We glad to see a new contributors!

Let's change this like following
```go
settings := telebot.Settings{
    Token:  sender.apiToken,
    Poller: &telebot.LongPoller{Timeout: pollerTimeout},
}
if sender.apiServer != "" {
    settings.URL = sender.apiServer
}
sender.bot, err := telebot.NewBot(settings)
```

Reply:
Hi @litleleprikon , thank you for your help. I'm not familiar with go, so guess your code more nice. I will change my pull request

Reply:
too many mistakes. i remove this request and will start again